config/samples/ibp.com_v1beta1_chaincodebuild_minio.yaml

```diff
@@ -15,6 +15,6 @@ spec:
       object: "go-contract"
     dockerBuild:
       pushSecret: "dockerhub-secret"
-      appImage: hyperledgerk8stest/go-contract
-      dockerfile: Dockerfile.optimized
-      context: bestchains/go-contract/samples/go-contract/
+      appImage: hyperledgerk8stest/go-contract-nonce
```

Comment:
now we use `basic` contract by default, so this `appImage` should be `go-contract-basic`